src/coreclr/gc/gc.cpp

```diff
@@ -25490,8 +25490,9 @@ bool gc_heap::change_heap_count (int new_n_heaps)
             from_heap_number = (from_heap_number + 1) % old_n_heaps;
         }
 
-        // prepare for the switch by fixing the allocation contexts on the old heaps,
+        // prepare for the switch by fixing the allocation contexts on the old heaps, unify the gen0_bricks_cleared flag,
         // and setting the survived size for the existing regions to their allocated size
+        BOOL unified_gen0_bricks_cleared = true;
```

Comment:
Capital `BOOL` should use `TRUE`?

Reply:
ahh yes, I changed it from bool to BOOL and didn't change this one.